### PR TITLE
Clean up Foreman 3.4 release notes

### PR DESCRIPTION
**Pull request by ekohl**
See individual commits for details.

The styles of authors are still quite different, but I wonder which is best. Perhaps a long list is not too bad given it's an appendix. Should Foreman adopt the same style as Katello?

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/doc-Release_Notes/topics/katello-contributors.adoc [hunk 1]
```diff
@@ -1,21 +1,21 @@
- Adam Růžička <adamruzicka@users.noreply.github.com>
```
Comment (maximiliankolb) [inline guides/doc-Release_Notes/topics/katello-contributors.adoc:1]: I assume this file was created automatically. Maybe for future releases, we should instead rework this script?

You change is fine for me though!

File: guides/doc-Release_Notes/topics/foreman-contributors.adoc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+We'd like to thank the following people who contributed to the Foreman {ProjectVersion} release:
+
```
Comment (maximiliankolb) [inline guides/doc-Release_Notes/topics/foreman-contributors.adoc:2]: How did you come up with this list?

File: guides/doc-Release_Notes/master.adoc [hunk 3]
```diff
@@ -10,6 +10,10 @@ ifdef::katello[]
 
 include::topics/foreman.adoc[leveloffset=+1]
 include::topics/katello.adoc[leveloffset=+1]
+endif::[]
+
+// Start inserting specific x.y.z releases here
+ifdef::katello[]
 include::topics/katello-4.6.0.adoc[leveloffset=+1]
```
Comment (maximiliankolb) [inline guides/doc-Release_Notes/master.adoc:17]: Maybe use an attribute for Katello version?

File: guides/doc-Release_Notes/topics/foreman-contributors.adoc (hunk 2 above)
Comment (ekohl) [reply guides/doc-Release_Notes/topics/foreman-contributors.adoc:2]: https://github.com/theforeman/theforeman.org/blob/gh-pages/_includes/manuals/3.4/1.2_release_notes.md#contributors

In https://github.com/theforeman/foreman-documentation/issues/1767 I kept notes on how I want to do this in this repository.

File: guides/doc-Release_Notes/master.adoc (hunk 3 above)
Comment (ekohl) [reply guides/doc-Release_Notes/master.adoc:17]: Are you suggesting this:
```suggestion
include::topics/katello-{KatelloVersion}.0.adoc[leveloffset=+1]
```

I'm not sure it helps that much. If you go that route, you could also consider naming the file `topic/katello-0.adoc` (and `topic/katello-1.adoc` for Katello 4.6.1). However, then we'd need to change it everywhere IMHO.

I'd prefer to merge this now and if you feel it's something we should do then we open an issue to refactor all branches.

File: guides/doc-Release_Notes/topics/katello-contributors.adoc (hunk 1 above)
Comment (ekohl) [reply guides/doc-Release_Notes/topics/katello-contributors.adoc:1]: I don't know how it was created, but https://github.com/theforeman/foreman-documentation/issues/1767 is open to unify it.

File: guides/doc-Release_Notes/master.adoc (hunk 3 above)
Comment (maximiliankolb) [reply guides/doc-Release_Notes/master.adoc:17]: OK, good point. Disregard this suggestion then.

**Review by maximiliankolb — APPROVED** [review]
I've resolved all open threads. LGTM.

**Review by evgeni — APPROVED** [review]

**Pull request merged**